apps/playground-web/src/lib/constants.ts

```diff
@@ -31,6 +31,7 @@ export const WALLETS = [
       passkeyDomain: getDomain(),
     },
   }),
+  ecosystemWallet("ecosystem.new-age"),
```

Comment:
also handled dev/prod in my PR (though commented)

we should make a new section in playground for ecosystem i reckon